packages/cache-inmemory/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@medusajs/cache-inmemory",
+  "version": "1.0.0",
+  "description": "In-memory Cache Module for Medusa",
+  "main": "dist/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/medusajs/medusa",
+    "directory": "packages/cache-inmemory"
+  },
+  "publishConfig": {
+    "access": "public",
+    "files": [
+      "dist"
+    ]
+  },
```

Comment:
question: `publishConfig` and `files` are typically added as two separate props at the root level. Can you verify that this works as expected? :)